lib/processes.js

```diff
@@ -97,6 +97,15 @@ function launchAllProcesses(config) {
         args.push('--headless');
       }
       if (process.getuid() === 0) args.push('--no-sandbox');
+
+      var debugPort = config.get('chrome.--remote-debugging-port');
```

Comment:
Any reason why this isn't the default? It feels weird that people have to opt-in to enable this feature..?

Reply:
Did you mean to always pass the port and set in the config? If yes, this was my initial thought -
In most cases, port number won't be used in the tests. Also, I was not very sure that, if we should always set the port even if it would be assigned by chromedriver.

But it makes sense now, if we are exposing API like _getPort_ from testium. We should make it return in consistent way

Reply:
IIRC there was some chat to just always read it from the port file written by Chrome. Didn't that work out?

Reply:
@jkrems Unfortunately, that didn't work. Not sure if that was due to system issue (some process errors out writing to hidden directory/files). Looking at [chromelauncher code](https://github.com/GoogleChrome/chrome-launcher/blob/master/src/chrome-launcher.ts#L235-L244), this option didn't seem bad to me :)

Reply:
So setting this option is compatible with `chromedriver` after all? If so - SGTM. Is there any way we can verify that Chrome has in fact been started with that option?

Reply:
Captured this SS from activity Monitor for the chrome instance. Port number generated programmatically was- `59196`. Not sure, if anything is obvious here :(

![screen shot 2018-07-02 at 6 14 09 pm](https://user-images.githubusercontent.com/33071225/42190717-55d1782e-7e24-11e8-94da-4e9466aa25e1.png)